### PR TITLE
Add netstandard1.1 to Microsoft.DiaSymReader targets

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@ericstj

File: src/Dependencies/Microsoft.DiaSymReader/Microsoft.DiaSymReader.nuspec [hunk 1]
```diff
@@ -5,10 +5,14 @@
     <summary>Microsoft DiaSymReader interop interfaces and utilities</summary>
     <description>
       Microsoft DiaSymReader interop interfaces and utilities.
-Supported Platforms:
-- .NET Framework 2.0
-- .NET Framework 4.5
     </description>  
+    <dependencies>
+      <group targetFramework="netstandard1.1">
+        <dependency id="System.Runtime" version="4.0.21-rc2-23910" />
```
Comment (ericstj) [inline src/Dependencies/Microsoft.DiaSymReader/Microsoft.DiaSymReader.nuspec]: Latest versions are 4.1.0-rc2-23910 for both System.Runtime and System.Runtime.InteropServices.

**Comment by tmat** [comment]
@agocke Please review.

**Comment by ericstj** [comment]
LGTM

**Pull request merged**